pkgs/development/tools/continuous-integration/buildkite-agent/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchurl, makeWrapper, coreutils, git, openssh, bash, gnused, gnugrep }:
+
+stdenv.mkDerivation rec {
+  name = "buildkite-agent-2.1.8";
```

Comment:
I encourage you to move the version string to a `version` attribute it and re-use it both in `name` and in `src.url`; this makes updating much more convenient.